tests/phpunit/tests/oembed/controller.php

```diff
@@ -279,6 +296,10 @@ function test_request_without_url_param() {
 		$this->assertSame( 'url', $data['data']['params'][0] );
 	}
 
+	/**
+	 * @covers WP_REST_Server::dispatch
+	 * @covers WP_HTTP_Requests_Response::get_data
```

Comment:
that is not what this covers. It covers this https://github.com/WordPress/wordpress-develop/blob/4b7b089896c966f61585db121242a1c79de425fa/src/wp-includes/class-wp-oembed-controller.php#L118-L138

Reply:
all of these need someone who understands the underlining function that are been tested